ajax/libs/coolprop/6.3.0/coolprop.js

```diff
@@ -0,0 +1,10661 @@
+// Copyright 2010 The Emscripten Authors.  All rights reserved.
```

Comment:
This file does not appear in the latest tagged release, 6.3.0: https://github.com/CoolProp/CoolProp/tree/v6.3.0/wrappers/Javascript